server/routes/hello.js

```diff
@@ -1,20 +1,56 @@
-module.exports = [
-  {
-    path: '/hello',
-    method: ['GET',],
-    config: {
-      auth: {
-        strategy: 'facebook',
-        mode: 'try',
-      },
-      handler: (request, reply) => {
-        if (!request.auth.isAuthenticated) {
-          return reply('Auth failed due to: ' + request.auth.error.message).code(401);
-        } else {
-          request.cookieAuth.set({user: request.auth.credentials.profile.id, });
-          reply.redirect('/');
+const querystring = require('querystring');
+const Request = require('request');
```

Comment:
either use lowercase or uppercase for module names. (I think you are using lower case)